src/tools/builder/dynamicwallpaperdescriptionreader.cpp

```diff
@@ -99,6 +99,12 @@ bool DynamicWallpaperDescriptionReaderPrivate::read(int index)
     const QJsonValue fileName = descriptor[QLatin1String("FileName")];
 
     QString absoluteFileName = fileName.toString();
+
+    if (absoluteFileName == "") {
```

Comment:
nit: use QString::isEmpty()

```suggestion
    if (absoluteFileName.isEmpty()) {
```